utils/sl/components/__init__.py

```diff
@@ -66,13 +66,6 @@ def check(self, name):
         self.not_installed.append(name)
         return False
 
-def exclude(reqs, name, use_value, exclude_value):
```

Comment:
If we remove this, do we also need to remove the EXCLUDE Variable?   Presently, it is only used for psm2 exclusion

Reply:
Possibly, you mean as a command line option?  I can't rule out needing this function again at some point so I was on the fence about leaving this in.

Reply:
yes, this implements the command line option by checking if psm2 is included on the command line.

Reply:
This bit isn't actually required for the rest of the functionality, I only included it as Johann had said that he thought we'd already remove it.

I did try testing with a different distro at one point, and psm2 doesn't compile there which is why it got removed, but that's no longer part of this PR.